user_templates/mcu_defaults/main/BTT_Manta_M8P.cfg

```diff
@@ -0,0 +1,46 @@
+#####################################################################
+#   BTT Manta M8P Pin Aliases
+#####################################################################
+[board_pins mcu_manufacturer]
+aliases:
+    MCU_MOTOR0_STEP=PE2    , MCU_MOTOR0_DIR=PB4    , MCU_MOTOR0_ENABLE=PC11   , MCU_MOTOR0_UART=PC10   ,
+    MCU_MOTOR1_STEP=PF12   , MCU_MOTOR1_DIR=PF11   , MCU_MOTOR1_ENABLE=PB3    , MCU_MOTOR1_UART=PF13   ,
+    MCU_MOTOR2_1_STEP=PD7  , MCU_MOTOR2_1_DIR=PD6  , MCU_MOTOR2_1_ENABLE=PF10 , MCU_MOTOR2_1_UART=PF9  ,
+    MCU_MOTOR3_STEP=PD3    , MCU_MOTOR3_DIR=PD2    , MCU_MOTOR3_ENABLE=PD5    , MCU_MOTOR3_UART=PD4    ,
+    MCU_MOTOR4_STEP=PC9    , MCU_MOTOR4_DIR=PC8    , MCU_MOTOR4_ENABLE=PD1    , MCU_MOTOR4_UART=PD0    ,
+    MCU_MOTOR5_STEP=PA10   , MCU_MOTOR5_DIR=PD15   , MCU_MOTOR5_ENABLE=PA15   , MCU_MOTOR5_UART=PF8    ,
+    MCU_MOTOR6_STEP=PD12   , MCU_MOTOR6_DIR=PD11   , MCU_MOTOR6_ENABLE=PD14   , MCU_MOTOR6_UART=PD13   ,
+    MCU_MOTOR7_STEP=PD10   , MCU_MOTOR7_DIR=PD8    , MCU_MOTOR7_ENABLE=PD9    , MCU_MOTOR7_UART=PC7    ,
+
+    MCU_STOP0=PF3  , MCU_STOP1=PF4  , MCU_STOP2=PF5 , MCU_STOP3=PC0 ,
+    MCU_STOP4=PC1  , MCU_STOP5=PC2  ,
+
+    MCU_HE0=PE3 , MCU_HE1=PB5 , MCU_HE2=PB6 , MCU_HE3=PE1 ,
+
+    MCU_BED0=PB7 ,
+
+    MCU_TB=PA0 ,
+    MCU_T0=PA1 , MCU_T1=PA2 , MCU_T2=PA3 , MCU_T3=PA4 ,
+
+    MCU_FAN0=PE6 , MCU_FAN1=PE0 , MCU_FAN2=PC12 , MCU_FAN3=PE5 , 
+    MCU_FAN4=PE4 , MCU_FAN4_TACH=PC13 ,
+    MCU_FAN5=PB8 , MCU_FAN5_TACH=PC14,
+    MCU_FAN6=PB9 , MCU_FAN6_TACH=PC15,
+
+    MCU_NEOPIXEL_1=PC6 , MCU_NEOPIXEL_2=PA9 ,
```

Comment:
Note that I renamed this pin, since M8P has 2 pins. Unsure what you want it to be named instead.

Reply:
I usually use what is written on the PCB directly or in board's documentation to be able to stick with the manufacturers names. But I'm really open to suggestion and this is ok to me if there is nothing specified :)